subworkflows/local/03_assemble/assemble_organelles.nf

```diff
@@ -1,40 +1,74 @@
-include { MITOHIFI_FINDMITOREFERENCE } from "../../../modules/nf-core/mitohifi/findmitoreference/main"
-include { MITOHIFI_MITOHIFI          } from "../../../modules/nf-core/mitohifi/mitohifi/main"
+include { MITOHIFI_FINDMITOREFERENCE                     } from "../../../modules/nf-core/mitohifi/findmitoreference/main"
+include { MITOHIFI_MITOHIFI as MITOHIFI_MITOHIFI_CONTIGS } from "../../../modules/nf-core/mitohifi/mitohifi/main"
+include { MITOHIFI_MITOHIFI as MITOHIFI_MITOHIFI_READS   } from "../../../modules/nf-core/mitohifi/mitohifi/main"
 
 workflow ASSEMBLE_ORGANELLES {
     take:
-    ch_raw_assemblies
+    ch_input
+    ch_input_mode
```

Comment:
Something I forgot about is that the pipeline is also propagating settings as a key in the meta map. Keep in mind that adding anything to this map though, means if it's value is changed ( e.g. exploring whether to use reads or contigs), it will invalidate the cache wherever this is part of the meta map. This is just a note. I'm not suggesting to use it in this case.